pkg/profilecreator/profilecreator_test.go

```diff
@@ -101,3 +102,42 @@ var _ = Describe("PerformanceProfileCreator: Getting Nodes from Must Gather", fu
 
 	})
 })
+
+var _ = Describe("PerformanceProfileCreator: Loading GHW Snapshot from Must Gather", func() {
+	var mustGatherDirPath, mustGatherDirAbsolutePath string
+	var node *v1.Node
+	var err error
+
+	Context("Identifying Nodes Info of the nodes cluster", func() {
+		It("gets the Nodes Info successfully", func() {
+			node = newTestNode("cnfd1-worker-0.fci1.kni.lab.eng.bos.redhat.com")
+			mustGatherDirPath = "../../testdata/must-gather/must-gather.local.directory"
+			mustGatherDirAbsolutePath, err = filepath.Abs(mustGatherDirPath)
+			Expect(err).ToNot(HaveOccurred())
+			snapShotOptions, err := LoadSnapshot(mustGatherDirAbsolutePath, node)
```

Comment:
does this require a `sysinfo.tgz` to be added to the source tree? If yes (and I'm not 100% happy with this decision) it should be added in this commit, otherwise the test is broken

Reply:
We have it already. I had added it in the the testdata in the previous PR as I knew we would be needing it here.

Reply:
ok, then I missed it :) sorry!